package.json

```diff
@@ -18,6 +18,7 @@
     "chartjs-adapter-moment": "^1.0.0",
     "debounce": "^1.2.1",
     "papaparse": "^5.3.2",
+    "sweetalert2": "11.4.8",
```

Comment:
this is not necessary. kindly remove it from here

Reply:
Why is not necessary ?

Reply:
What would be the solution for then in Dokan Admin panel sweetalert2 popups?

Reply:
we are not using any separate library for vue, we are using the default one. this is not necessary because we don't want to include same library twice.

Reply:
beside including any libraries in the required section will increase the bundle size which is already way to high than the recommended value.

Reply:
I'm approving this pr for now, since this code was added long ago. If you are curious, kindly check the product advertisement module code and check how I've implemented it without adding this library to bootstrap.